src/extension/exponent/exponentHelper.ts

```diff
@@ -30,12 +33,15 @@ const DEFAULT_ANDROID_INDEX = "index.android.js";
 
 const DBL_SLASHES = /\\/g;
 
+const NGROK_PACKAGE = "@expo/ngrok";
+
 export class ExponentHelper {
     private workspaceRootPath: string;
     private projectRootPath: string;
     private fs: FileSystem;
     private hasInitialized: boolean;
     private logger: OutputChannelLogger = OutputChannelLogger.getMainChannel();
+    private nodeModulesGlobalPathAddedToEnv: boolean;
```

Comment:
Could you please declare `nodeModulesGlobalPathAddedToEnv` variable before optional parameters?